Products/CMFPlone/tests/testEmailLogin.py

```diff
@@ -83,8 +83,8 @@ def testEmailMemberIdsAllowed(self):
         # email addresses.
         # A plus sign in the id gives problems in some parts of the
         # UI, so we do not allow it.
-        self.assertFalse(pattern.match("user+test@example.org"))
-        self.assertFalse(registration.isMemberIdAllowed("user+test@example.org"))
+        #self.assertFalse(pattern.match("user+test@example.org")) 
+        #self.assertFalse(registration.isMemberIdAllowed("user+test@example.org"))
```

Comment:
Can you change this so it tests that these email address *are* allowed?  So change `assertFalse` in `assertTrue`.  And then the comment can be removed.

Reply:
please verify @mauritsvanrees